test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -407,7 +405,7 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			}
 		})
 
-		It("Reverts back all profile configuration", func() {
+		AfterAll(func() {
```

Comment:
Setup Node not in Ordered Container
AfterAll(func() {
/go/src/github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go:408
  [AfterAll] setup nodes must appear inside an Ordered container.  They cannot
  be nested within other containers, even containers in an ordered container.